components/drivers/misc/adc.c

```diff
@@ -205,7 +233,7 @@ static int adc(int argc, char **argv)
                 }
                 else
                 {
-                    rt_kprintf("adc read <channel>     - read adc value on the channel\n");
+                   rt_kprintf("adc read <channel>     - read adc value on the channel\n");
```

Comment:
The space is not correct, the original is correct. This line shouldn't be changed